supplier/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+
+class User(models.Model):
+    user_name = models.CharField(max_length=32,
+                                 primary_key=True,
+                                 null=False)
+    first_name = models.CharField(max_length=32)
+    last_name = models.CharField(max_length=32)
+    password = models.CharField(max_length=16)
+    user_type = models.CharField(max_length=16)
```

Comment:
min validators should be included in all

Reply:
for first and last named, i gave the client an option for null.